EarlGrey/Core/GREYKeyboard.m

```diff
@@ -36,9 +37,9 @@
 static GREYTapAction *gTapKeyAction;
 
 /**
- *  Flag set to @c YES when the keyboard is shown, @c NO when keyboard is hidden.
+ *  Flag set to @c true when the keyboard is shown, @c false when keyboard is hidden.
  */
-static BOOL gIsKeyboardShown = NO;
+static atomic_bool gIsKeyboardShown = false;
```

Comment:
[Here's](http://stackoverflow.com/questions/31535840/which-scenes-keyword-volatile-is-needed-to-declare-in-objective-c) also a good read on `volatile`.